index.js

```diff
@@ -53,5 +41,12 @@ module.exports = function nuxtApollo(moduleOptions) {
       delete gqlRules.exclude
     }
     config.module.rules.push(gqlRules)
+    if (isServer) {
```

Comment:
@kieusonlam what is nodeExternals whitelist actually doing (I never used it so far)

Reply:
I removed it on https://github.com/nuxt-community/apollo-module/pull/106/commits/8fce5b3c6d15a290316586add499a833d5e12009